srcpkgs/write-styluslabs/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'write-styluslabs'
+pkgname=write-styluslabs
+version=3.0.0
+revision=1
+archs="x86_64"
+wrksrc="Write"
+depends="SDL2"
+short_desc="Word processor for handwriting"
+maintainer="Cat Marin <me@jmarin.xyz>"
+license="custom:Proprietary"
```

Comment:
As the `xlint` CI test shows, a proprietary license declaration requires use of `vlicense` in `do_install` or `post_install` to copy the license text into the package. As it stands, I see no license terms attached to this software. You should identify the license terms and be able to fetch the license text from somewhere so we know what terms govern the software.

Reply:
I'm currently in the process of contacting Stylus Labs on how they'd like me to proceed here, I'll update my PR when I know.

Reply:
If they don't make versioned releases. it's unlikely to be accepted as a package here.

Reply:
I'm not sure how well it fits under quality requirements either.

Reply:
> If they don't make versioned releases. it's unlikely to be accepted as a package here.

I've sent them an email regarding the issues on their end (including the license), I'll be closing my pull request until they respond/provide versioned downloads and a license.

Reply:
http://www.styluslabs.com/write/write300.tar.gz